modules/aerodyn/src/AeroDyn_Types.f90

```diff
@@ -188,7 +189,7 @@ MODULE AeroDyn_Types
 ! =========  AD_InputFile  =======
   TYPE, PUBLIC :: AD_InputFile
     LOGICAL  :: Echo      !< Echo input file to echo file [-]
-    REAL(DbKi)  :: DTAero      !< Time interval for aerodynamic calculations {or "default"} [s]
+    REAL(DbKi)  :: DTAero      !< Time interval for aerodynamic calculations {or  [default"}"]
```

Comment:
The comments/units here are a little bit off... probably due to the way quotation marks are used in the registry input file.